src/components/entry/EntryDetails.js

```diff
@@ -1,19 +1,239 @@
+            />
+          )}
+        </HeaderContainer>
+        <TagContainer>
+          {locationData.access && <Tag>{ACCESS_TYPE[locationData.access]}</Tag>}
+          <Tag color="#4183C4" backgroundColor="#D9E6F3">
+            {locationData.unverified ? 'Unverified' : 'Verified'}
+          </Tag>
+        </TagContainer>
+        <DescriptionContainer>
+          <Description>{locationData.description}</Description>
+
+          <UpdateText>
+            Last Updated {parseISOString(locationTypeData.updated_at)}
+          </UpdateText>
+
+          <ButtonSpacing>
+            <Button>
+              <Star /> Review
+            </Button>
+            <Button secondary>
+              <Flag /> Report
+            </Button>
+          </ButtonSpacing>
+        </DescriptionContainer>
+
+        <ResourceHeader>Other Resources</ResourceHeader>
+
+        {locationTypeData.usda_symbol && (
+          <IndividualResourceContainer>
+            <img src={USDALogo} height="20px" width="25px" alt="USDA logo" />
+            <a href={locationTypeData.usda_symbol}>USDA</a>
+          </IndividualResourceContainer>
+        )}
+
+        {locationTypeData.wikipedia_url && (
+          <IndividualResourceContainer>
+            <Wikipedia height="20px" width="25px" />
+            <a href={locationTypeData.wikipedia_url}>Wikipedia</a>
+          </IndividualResourceContainer>
+        )}
+
+        {locationTypeData.eat_the_weeds_url && (
+          <IndividualResourceContainer>
+            <img
+              src={EatTheWeedsLogo}
+              height="20px"
+              width="25px"
+              alt="Eat the Weeds logo"
+            />
+            <a href={locationTypeData.eat_the_weeds_url}>Eat the Weeds</a>
+          </IndividualResourceContainer>
+        )}
+
+        {locationTypeData.foraging_texas_url && (
+          <IndividualResourceContainer>
+            <img
+              src={ForagingTexasLogo}
+              height="20px"
+              width="25px"
+              alt="Foraging Texas logo"
+            />
+            <a href={locationTypeData.foraging_texas_url}>Foraging Texas</a>
+          </IndividualResourceContainer>
+        )}
+
+        {locationTypeData.urban_mushrooms_url && (
+          <IndividualResourceContainer>
+            <img
+              src={UrbanMushroomsLogo}
+              height="20px"
+              width="25px"
+              alt="Urban Mushrooms logo"
+            />
+            <a href={locationTypeData.urban_mushrooms_url}>Urban Mushrooms</a>
+          </IndividualResourceContainer>
+        )}
+
+        {locationTypeData.fruitipedia_url && (
+          <IndividualResourceContainer>
+            <img
+              src={FruitipediaLogo}
+              height="20px"
+              width="25px"
+              alt="Fruitipedia logo"
+            />
+            <a href={locationTypeData.fruitipedia_url}>Fruitipedia</a>
+          </IndividualResourceContainer>
+        )}
```

Comment:
Noticed you have a lot of code repetition here, from lines 170-230. Could you try to convert this into a loop?